odk1-src/collect-settings.rst

```diff
@@ -209,9 +210,7 @@ To access Form Management settings:
 
 :guilabel:`Default to finalized`
 """""""""""""""""""""""""""""""""
-  When enabled, forms are automatically finalized
-  upon reaching the end of the form.
-  You can opt out of this on any specific form during form completion.
+  When enabled, records are set to be finalized when saved at the end of a form-filling session. You can opt out of this at the end of filling any specific record. This is particularly important to consider when forms are encrypted because encryption happens on finalization. Finalized records for encrypted forms can't be opened because they are encrypted. Records for encrypted forms that have not been finalized are not encrypted and can be edited.
```

Comment:
It'd be nice to make "forms are encrypted" be a link to the encrypted forms doc.

Reply:
Thank you, great call!